paper/paper.md

```diff
@@ -28,28 +28,28 @@ bibliography: paper.bib
+Seaduck provides highly-customizable interpolation methods for both Eulerian and Lagrangian points. Users can control all the properties of a hierarchy of kernels, including: (1) The shape of the interpolation kernel(s) in both spatial and temporal dimensions, which defines which neighboring points are used, and therefore how the continuous field is estimated. (2) The interpolation weight function, which allows users to calculate generic linear operations on the data, such as differentiation and smoothing, in all four dimensions. The hierarchy of kernels controls behaviour near land-masked points. Specifically, the hierarchy consists of successively more compact kernels that are used depending on the proximity of land points.
 
-With the above suite of functionalities, seaduck is capable of accomplishing many common tasks in ocean dataset analysis including interpolation, regridding and Lagrangian particle simulation and some new ones including interpolation in Lagrangian label space and analyzing tracer budget from Lagrangian perspective. We also strive to make seaduck an accessible education tool by creating a very simple high-level interface intended for people with little programming background.
+With the above functionality, seaduck can accomplish many common tasks in ocean model data analysis, including interpolation, regridding, and Lagrangian particle simulation. Less common tasks are also possible, such as interpolation in Lagrangian label space, and analysis of tracer budgets along Lagrangian trajectories. We also strive to make seaduck an accessible education tool by creating a very simple high-level default interface, which is intended for people with little programming background, and for people who want to quickly try the tool.
 
 # Usage Examples
 
-While some usage examples are presented here, many more can be found in the documentation for seaduck (https://macekuailv.github.io/seaduck/). The notebooks  of the following examples run on SciServer[@sciserver], an openly available cloud compute resource. A supplementary GitHub repository (https://github.com/MaceKuailv/seaduck_sciserver_notebook) holds all SciServer notebooks, which is being continuously maintained.
+While some usage examples are presented here, many more can be found in the documentation for seaduck (https://macekuailv.github.io/seaduck/). The notebooks  of the following examples run on SciServer [@sciserver], an openly available cloud compute resource for scientific data analysis. A supplementary GitHub repository (https://github.com/MaceKuailv/seaduck_sciserver_notebook) holds all SciServer notebooks, and is being continuously maintained.
 
-![Fig.1 (a) Scatterplot with colors showing the sea surface height value near Kangerdlugssuaq Fjord defined in the model and interpolated by seaduck.\label{fig:onlyone}. (b) Streaklines of particle advected by stationary 2D slice of the LLC4320 simulation, colors denotes the current speed.](fig1.png)
```

Comment:
Split the two panels into two separate figures so they're larger and easier to read.